protocol/avdl/keybase1/chat_local.avdl

```diff
@@ -73,6 +73,7 @@ protocol chatLocal {
   }
 
   record ConversationInfoLocal {
+    chat1.ConversationID id;
```

Comment:
we should really use a `variant` for this.  CAn you rewrite it once I get my fix in?